### ISSUE TITLE
CLient ciphers for OpenSSL1.0.2 incorrect

### ISSUE THREAD

**Issue by seccubus** [ssllabs/ssllabs-scan#298]
Ciphers 0xb and 0xe don't belong here: https://www.ssllabs.com/ssltest/viewClient.html?name=OpenSSL&version=1.0.2

TLS_DH_RSA_EXPORT_WITH_DES40_CBC_SHA (0xe)   INSECURE	40
TLS_DH_DSS_EXPORT_WITH_DES40_CBC_SHA (0xb)   INSECURE	40

From OpenSSL change log:
```
) Removed support for the two export grade static DH ciphersuites
EXP-DH-RSA-DES-CBC-SHA and EXP-DH-DSS-DES-CBC-SHA. These two ciphersuites
were newly added (along with a number of other static DH ciphersuites) to
1.0.2. However the two export ones have *never worked since they were
introduced. It seems strange in any case to be adding new export
ciphersuites, and given "logjam" it also does not seem correct to fix them.
[Matt Caswell]
```

Thanks to @drwetter for figuring this one out.

**Comment by SkateScout** [comment]
Please add also the patch level for 1.0.1 "L" is mentioned in (sadly in Lower case I or l problem)
But for 1.0.2 there is no patch level mentioned.

**Comment by seccubus** [comment]
Got word from @petermosmans that these two ciphers will be reenabled.

**Comment by ivanr** [comment]
Fixed in 1.22.8, now running in dev. The old (probably pre-release) 1.0.2 simulation has been replaced with the latest 1.0.2e.
